### PR TITLE
Feature/implement functions and logics for countdown clock

### PR DESCRIPTION
**Pull request by nddmanh**
Time on client UI countdown from 59 to 0:

image

image

### CODE REVIEW

**Comment by ppppbn** [comment]
I think we should use a clock as UI. not a div like this

File: src/modules/playRoom/pages/PlayRoom.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.js:1]: no need to import React

File: src/modules/playRoom/components/CountDown.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState }  from 'react';
 
 const CountDown = () => {
+  const [countDown, setCountDown] = useState(59);
+
+  useEffect(() => {
```
Comment (ppppbn) [inline src/modules/playRoom/components/CountDown.js:6]: as I said earlier, we won't let the count down happens on front end side. Instead we should handle the event sent from server to update the clock value

File: src/modules/playRoom/components/CountDown.js [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
+    }, 1000);
+
+    if (countDown === -1) {
+      setCountDown(59) ;
+    }
+
+    return () => {
+      clearInterval(time);
+    };
+  }, [countDown]);
```
Comment (ppppbn) [inline src/modules/playRoom/components/CountDown.js:18]: Will this cause infinite loop for useEffect?

File: src/modules/playRoom/components/ChatBox.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/ChatBox.js:1]: Remove React

File: src/modules/playRoom/components/CountDown.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState }  from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/CountDown.js:1]: Remove React

File: src/modules/playRoom/components/PickItem.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:1]: Remove React

File: src/modules/playRoom/components/PickItem.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:22]:
```suggestion
      <div className="select" onClick={showModal} >                
```

File: src/modules/playRoom/components/PickList.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:1]: Remove React

File: src/modules/playRoom/pages/PlayRoom.css [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+    position: absolute;
+    bottom: 0;
+}
+
+.chat-input input {
+    width: calc(100% - 50px);
+    height: 100%;
+    background-color: #4f5d7321;
+    border-radius: 50px;
+    color: white;
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:150]: Use hex instead

File: src/modules/playRoom/pages/PlayRoom.css [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+    justify-content: start;
+    align-items: center;
+    height: 40px;
+    background-color: #77b3d4;
+    padding: 15px;
+}
+.head-result>h3 {
+    margin-top: 10px;
+    font-size: 16px;
+    color: #fff;
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:217]: You should use hex code with same style 3 characters or full code like this #ffffff

Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:217]:
```suggestion
    color: #ffffff;
```

File: src/modules/playRoom/pages/PlayRoom.css [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+}
+
+.client-info {
+    color: #fff;
+    padding: 5px;
+    margin-top: 15px;
+}
+.client-info > h2 {
+    font-weight: 650;
+    color: white;
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:130]: Use hex code

File: src/modules/playRoom/pages/PlayRoom.css [hunk 12]
```diff
@@ -0,0 +1,236 @@
...
+    background-color: #8bbeda;
+    cursor: pointer;
+}
+
+/* chat section */
+
+.chats {
+    width: 100%;
+    padding: 0 15px;
+    color: #fff;
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:82]:
```suggestion
    color: #ffffff;
```

File: src/modules/playRoom/pages/PlayRoom.css [hunk 13]
```diff
@@ -0,0 +1,236 @@
...
+    position: relative;
+}
+
+/* scroll bar  */
+.scroll {
+    width: 100%;
+    height: 74%;
+    overflow: hidden;
+    overflow-y: scroll;
+    /* overflow-y: scroll; */
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:59]: Remove this comment

File: src/modules/playRoom/components/ResultRoll.js [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+  ]);
+  return (
+    <div className="previous-results">
+      <div className="head-result">
+        <h3>KẾT QUẢ</h3>
+      </div>
+      <div className="result">
+        <div className="result-box">
+          {imagesResultRoll.map((result) => (
+            <input type="image" className='img-result' src={result.src} alt="" name="" value={result.value} />
```
Comment (lethemanh) [inline src/modules/playRoom/components/ResultRoll.js:45]:
```suggestion
            <input type="image" className="img-result" src={result.src} alt="image description" value={result.value} />
```

**Pull request closed without merging**